gapic-generator-java/src/test/proto/echo.proto

```diff
@@ -140,6 +140,16 @@ service Echo {
       body: "*"
     };
   }
+
+  // This is similar to the PagedExpand except that it uses
+  // max_results instead of page_size, as some legacy APIs still
+  // do. New APIs should NOT use this pattern.
+  rpc PagedExpandLegacy(PagedExpandLegacyRequest) returns (PagedExpandResponse) {
```

Comment:
I don't think this test can prove much because the package is not bigquery, this is more like a negative test case which is already covered by other test cases. We need a positive golden test file that can demonstrate the changes.

Reply:
SGTM, thanks!